api/src/main/java/pl/kolendateam/dadcard/classCharacter/entity/ClassPc.java

```diff
@@ -22,7 +22,10 @@ public class ClassPc implements Serializable {
     double classBab;
 
     public void incrementLevel() {
-        this.level += 1;
+        this.level++;
```

Comment:
ciekawostka: jest suptelna różnica miedzy ++i a i++ https://www.geeksforgeeks.org/what-is-the-difference-between-i-and-i-in-java/